sdk/cognitivelanguage/test-resources.bicep

```diff
@@ -0,0 +1,156 @@
+param baseName string = resourceGroup().name
+param location string = resourceGroup().location
+param testApplicationOid string
+param enableVersioning bool = false
+
+var taRoleId = 'a97b65f3-24c7-4388-baec-2e87135dc908'
+var cognitiveAccountName = 'textanalytics-${baseName}'
+var cognitiveApiVersion = '2024-04-01-preview'
+var storageAccountName = baseName
+var blobDataContributorRoleId = 'ba92f5b4-2d11-453d-a403-e96b0029c9fe'
+var blobDataOwnerRoleId = 'b7e6dc6d-f1e8-4753-8033-0f276bb0955b'
+var encryption = {
+  services: {
+    file: {
+      enabled: true
+    }
+    blob: {
+      enabled: true
+    }
+  }
+  keySource: 'Microsoft.Storage'
+}
+var networkAcls = {
+  bypass: 'AzureServices'
+  virtualNetworkRules: []
+  ipRules: []
+  defaultAction: 'Allow'
+}
+var containerName = 'documents'
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2023-05-01' = {
```

Comment:
Not a blocker, I was just curious is there a way to format string so we can get the API version stored in a variable instead?

Reply:
Nope, it has to be a string literal

Reply:
That makes sense. I saw we can do this in ARM and was curious.